src/main/resources/testdata.properties

```diff
@@ -0,0 +1,8 @@
+key=8fd60c025c90ad06311235cb57e9fb49
```

Comment:
you'd better separate credentials and test data. We usually add configurations in .properties files. Even better - don't store credentials in GIT

Reply:
Fixed.